src/app/screens/Home/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import browser from "webextension-polyfill";
+import { CashIcon, CogIcon } from "@heroicons/react/outline";
+
+import utils from "../../../common/lib/utils";
+import { getFiatFromSatoshi } from "../../../common/utils/helpers";
+import Transactions from "../../components/Transactions";
+import Loading from "../../components/Loading";
+import BalanceCard from "../../components/BalanceCard";
+
+class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      alias: "",
+      currency: "USD",
+      balance: null,
+      balanceFiat: null,
+      transactions: {},
+      loadingTransactions: true,
+    };
+  }
+  get exchangeRate() {
+    if (!this.state.balance) return null;
+    return (this.state.balanceFiat / this.state.balance) * 100000000;
+  }
+
+  componentDidMount() {
+    browser.storage.sync.get(["currentAccount"]).then((result) => {
+      this.setState({ currentAccount: result.currentAccount });
+    });
+    utils.call("getInfo").then((info) => {
+      console.log("info", info);
+      this.setState({ alias: info?.alias });
+    });
+    utils.call("getTransactions").then((result) => {
+      console.log(result);
+      this.setState({
+        transactions: result?.payments,
+        loadingTransactions: false,
+      });
+    });
+    utils.call("getBalance").then(async (result) => {
+      this.setState({ balance: result?.balance });
+      this.setState({
+        balanceFiat: await getFiatFromSatoshi(
+          this.state.currency,
+          result.balance
+        ),
+      });
+    });
+  }
+
+  render() {
+    const { alias, balance, balanceFiat, transactions } = this.state;
+
+    return (
+      <div>
+        <div className="px-5 space-x-4 py-2 flex items-center border-b border-gray-100">
+          <CashIcon className="h-6 w-6 text-gray-400" aria-hidden="true" />
+          <div className="flex-auto">
+            <div className="text-xs text-gray-500">{alias || "\u00A0"}</div>
+            <div className="text-xs">₿0.0016 7930 €33.57</div>
+          </div>
+          <button
+            className="ml-auto"
+            onClick={() => {
+              return utils.openPage("options.html");
+            }}
+          >
+            <CogIcon className="h-6 w-6 text-gray-400" aria-hidden="true" />
+          </button>
+        </div>
+        <div className="p-5 border-b-4 border-gray-200">
+          <BalanceCard
+            alias="Wallet name"
+            crypto={`₿${balance || ""}`}
+            fiat={`$${balanceFiat || ""}`}
+          />
+        </div>
+        <div className="p-5">
+          <h2 className="text-xl">Transactions</h2>
+          {this.state.loadingTransactions ? (
+            <div className="pt-4 flex justify-center">
+              <Loading />
+            </div>
+          ) : (
+            <Transactions
+              exchangeRate={this.exchangeRate}
```

Comment:
later we will have the exchange rate on the individual transaction level. (the rate at the time of the transaction)